### PR TITLE
Neue 6.x

### PR DESCRIPTION
**Pull request by DFurnes**
# Changes
- Updates SCSS and template markup to be compatible with Neue 6.0. Yikes. :japanese_ogre: 
- Using Ruby Sass temporarily while Libsass catches up.

For review: @weerd @sbsmith86

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_taxonomy.scss [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+//
+//  .container__body,
+//  .additional-text {
+//    p,
+//    ul,
+//    ol {
+//      @include compacted();
+//    }
+//  }
+  // @TODO ^ What was this?
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_taxonomy.scss]: The compacted class was added to each individual element to make it visually narrower, but I think this was just a temp fix...?

**Comment by sbsmith86** [comment]
Looks good to me :v:

**Comment by weerd** [comment]
NO.

**Comment by DFurnes** [comment]
Rebased against dev, and merging. :facepunch:

**Pull request merged**